api/v1/submissions/PKPSubmissionController.php

```diff
@@ -1273,6 +1273,11 @@ public function editPublication(Request $illuminateRequest): JsonResponse
             return response()->json($errors, Response::HTTP_BAD_REQUEST);
         }
 
+        // If no issue is selected, it will be considered as continuous publication
```

Comment:
If possible, any references to `issueId` should be kept in the OJS codebase. (There are a few places where we aren't 100% tidy with this.)